tests/jballerina-unit-test/src/test/resources/test-src/expressions/varref/record-variable-reference.bal

```diff
@@ -369,6 +369,57 @@ function testMappingBindingWithSingleNameFieldBinding() {
     assertEquality("bar4", b);
 }
 
+function testMappingBindingPatternInListBindingPattern() {
```

Comment:
Name is no longer accurate?

Reply:
Let's also update the name to reflect the actual test scenario. i.e., when used with an open record. The name is too vague atm.